installing/installing_ibm_power/creating-ibm-power-vs-workspace.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+[id="creating-ibm-power-vs-workspace"]
+= Creating an {ibmpowerProductName} Virtual Server workspace
+include::_attributes/common-attributes.adoc[]
+:context: creating-ibm-power-vs-workspace
+
+:FeatureName: {ibmpowerProductName} Virtual Server using installer-provisioned infrastructure
+include::snippets/technology-preview.adoc[]
+
+include::modules/creating-ibm-power-vs-workspace-procedure.adoc[leveloffset=+1]
+
+[id="next-steps_creating-ibm-power-vs-workspace"]
```

Comment:
Something is wrong with this heading on the preview: https://47558--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/creating-ibm-power-vs-workspace.html#creating-ibm-power-vs-workspace-procedure_creating-ibm-power-vs-workspace

Reply:
Have removed the heading from the module. As the heading of the assembly explains the task. 
@stevsmit request you to please review. Thanks.

Reply:
@stevsmit updated the heading. Request you to please check. Thanks.